inception/inception-api-annotation-experimental/src/main/java/de/tudarmstadt/ukp/inception/experimental/api/messages/response/relation/SelectRelationResponse.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed to the Technische Universität Darmstadt under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The Technische Universität Darmstadt
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.experimental.api.messages.response.relation;
+
+import java.util.List;
+
+import de.tudarmstadt.ukp.clarin.webanno.api.annotation.model.VID;
+
+public class SelectRelationResponse
+{
+    private VID relationAddress;
+    private VID governorId;
+    private String governorCoveredText;
+    private VID dependentId;
+    private String dependentCoveredText;
+    private String color;
+    private String type;
+    private List<String> features;
+
+    public SelectRelationResponse(VID aRelationAddress, VID aGovernorID, String aGovernorCoveredText,
```

Comment:
What does it mean to "select" a relation? Does it mean that the relation details are opened in the right sidebar (annotation editor detail panel)? If not, what does it mean?